modules/argocd/Makefile

```diff
@@ -1,15 +1,31 @@
-RELEASE_TAG ?= 0.1.0
-.PHONY: updater/test check-prereq-variables
+RELEASE_TAG ?= 1.0.0
+dirs ?= rendered/
+envs ?=
+apps ?= ""
```

Comment:
In `make tanka/generate` these are `env` and `app` (singular) even though you can specify multiple.

Reply:
(accidentally started a review when I meant to respond to this comment), sure I updated to match the tanka/generate format, but this is a space separated list rather than comma separated. this is reflected in the confluence docs